practiceCourseGenerator/README.md

```diff
@@ -0,0 +1,69 @@
+# Canvas Practice Course Generator
+
+Description: Python script for generating zip file that can be used to create 
+practice courses for students in Canvas.
+
+Installation:
+
+For this script to run the following needs to be installed...
+
+1. Python 2.7
+2. google-api-python-client
+
+Installing the google-api-python-client is simple with PIP:
+
+In a command line teminal, type the following:
+
+	pip install --upgrade google-api-python-client
+
+Of course this requires that you have PIP installed.
+
+To install PIP open a terminal, type the following:
+
+	sudo easy_install pip
+
+Mac's come with Python 2.7 which includes easy_install so this should work.
+
+
+Input: 
+
+1. input.csv - gathered from Google Drive
+2. propertiesProd.json -  holds custom properties
+3. storage.json - holds information to access google drive
+4. tl_client_secret.json - holds information to access google drive
+5. token.txt -  holds token to use Canvas API
+
+Output: 
+
+1. Directory - canvas_files_[Date]
```

Comment:
I'd add a comment here that this directory is assumed to be used as input to the SIS_import script